crates/biome_configuration/src/javascript/formatter.rs

```diff
@@ -21,8 +21,16 @@ pub struct JavascriptFormatter {
     pub quote_properties: QuoteProperties,
 
     /// Print trailing commas wherever possible in multi-line comma-separated syntactic structures. Defaults to "all".
-    #[partial(bpaf(long("trailing-comma"), argument("all|es5|none"), optional))]
-    pub trailing_comma: TrailingComma,
+    #[partial(bpaf(
+        long("trailing-commas"),
+        // TODO: Remove in 2.0.0
+        long("trailing-comma"),
+        argument("all|es5|none"),
+        optional
+    ))]
+    // TODO: Remove in 2.0.0
+    #[partial(serde(alias = "trailingComma"))]
+    pub trailing_commas: TrailingCommas,
```

Comment:
This should be supported by [serde](https://serde.rs/field-attrs.html#alias). But deserialization from a configuration file with the field `trailingComma` still generates a diagnostic.